### DISCUSSION TITLE
How to run webrtc-streamer without black window?

### DISCUSSION THREAD

**Discussion by miaomiaopan** [mpromonet/webrtc-streamer#721] [Q&A] [answered]
Hi, guys, how to run webrtc-streamer without black window? I use /B in bat file, but it doesn't work, thank you much.

**Comment by mpromonet** [comment, marked as answer]
Hi @miaomiaopan 

This may help https://stackoverflow.com/questions/415409/run-batch-file-as-a-windows-service 
nssm seems interesting way:
```
choco install nssm
nssm install webrtc-streamer <path>\webrtc-streamer
nssm start webrtc-streamer
```

Best Regards,
Michel.

**Reply by miaomiaopan** [reply]
Yes, It works, Thank you for your reply.

**Reply by miaomiaopan** [reply]
> Hi @miaomiaopan
> 
> This may help https://stackoverflow.com/questions/415409/run-batch-file-as-a-windows-service nssm seems interesting way:
> 
> ```
> choco install nssm
> nssm install webrtc-streamer <path>\webrtc-streamer
> nssm start webrtc-streamer
> ```
> 
> Best Regards, Michel.

Yes, It works, Thank you for your reply.
